charts/kueue/README.md

```diff
@@ -49,7 +49,7 @@ The following table lists the configurable parameters of the kueue chart and the
 | `enablePrometheus`                                     | enable Prometheus                                      | `false`                                     |
 | `enableCertManager`                                    | enable CertManager                                     | `false`                                     |
 | `controllerManager.kubeRbacProxy.image`                | controllerManager.kubeRbacProxy's image                | `gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0` |
-| `controllerManager.manager.image`                      | controllerManager.manager's image                      | `gcr.io/k8s-staging-kueue/kueue:main`       |
+| `controllerManager.manager.image`                      | controllerManager.manager's image                      | `us-central1-docker.pkg.dev/k8s-staging-images/kueue:main`       |
```

Comment:
Can we make sure that the staging registries never appear in an end-user facing doc/helm chart? This should be `registry.k8s.io/kueue/kueue`

Reply:
I think it's possible to have only once `kueue` in the name, isn't? Or should we have `kueue/kueue`?

Reply:
If someone is trying to use the charts from source, they currently point to the staging repo, so that they can get the latest build.
Alternatively, they can use the released chart https://github.com/kubernetes-sigs/kueue/releases/tag/v0.8.0, which points to registry.k8s.io

Reply:
As I understand, having `kueue/kueue` creates a `kueue` subdirectory in the registry and I think we can have the images in the root using only `us-central1-docker.pkg.dev/k8s-staging-images/kueue`, WDYT?

Reply:
`us-central1-docker.pkg.dev/k8s-staging-images/kueue` isn't an image but a registry.

Google changed the format of AR registries.

Reply:
I mean that as long as the image name matches the registry's, it is possible to store the image. So we can use the registry name as image name without adding an extra `kueue` and just reference the image as `us-central1-docker.pkg.dev/k8s-staging-images/kueue:latest`.

Reply:
Oh, I understand now. I agree with @upodroid.
If `us-central1-docker.pkg.dev/k8s-staging-images/kueue` is a registry, we can't use that as an image name too. The image name has to be `us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue:labelname`

Reply:
You can publish an image with the same name as the registry and retain the existing behaviour.

Reply:
Even if it works.... it's going to make this folder look like a mess if other projects decide to do the same
![image](https://github.com/user-attachments/assets/8489c6ef-c7b9-4edc-a4a7-bb083fd95512)

Better just add an extra /kueue... it's just for staging anyways.

Reply:
> Even if it works.... it's going to make this folder look like a mess if other projects decide to do the same ![image](https://private-user-images.githubusercontent.com/1299064/350520652-8489c6ef-c7b9-4edc-a4a7-bb083fd95512.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE0MTU1NjcsIm5iZiI6MTcyMTQxNTI2NywicGF0aCI6Ii8xMjk5MDY0LzM1MDUyMDY1Mi04NDg5YzZlZi1jN2I5LTRlZGMtYTRhNy1iYjA4M2ZkOTU1MTIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTlUMTg1NDI3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YjI3NWNmYjk4Nzg4Y2I3Mzc3N2IxZmFjZTc3NTkzOTg4ZTY5ZjY1ZTQ2NzkzMDBmOTUyMWY3NzJiZjJjYTdkYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.GHASgLO18K-ZOHa1WsihRMd0xCKrslSHtIAxdFQbwV4)
> 
> Better just add an extra /kueue... it's just for staging anyways.

I think the parent directory wouldn't change. Keeping only one `kueue` in the name will still store the images in the kueue directory. It would look like this 

<img width="777" alt="Screenshot 2024-07-19 at 20 57 04" src="https://github.com/user-attachments/assets/e886260b-f301-48ee-9f1f-6f410ed4206f">

Adding an extra /kueue will just create a subdirectory inside of the existing kueue directory.

Reply:
uhmm, I see. Ok then.

Reply:
That is expected. I just pushed a prod image to the kueue registry repo to demonstrate this.

https://console.cloud.google.com/artifacts/docker/k8s-staging-images/us-central1/kueue

```
 mahamed  MAHAALI-M-2PY9  ~  Desktop  Git  $  gcrane cp registry.k8s.io/kueue/kueue:v0.7.1 us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue:v0.7.1
2024/07/19 22:21:00 Copying from registry.k8s.io/kueue/kueue:v0.7.1 to us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue:v0.7.1
2024/07/19 22:21:07 pushed blob: sha256:e8d9a567199d7a318c875f2558a679ba8a924f817afacbb428afc3ffe6be6828
2024/07/19 22:21:07 pushed blob: sha256:b6824ed73363f94b3b2b44084c51c31bc32af77a96861d49e16f91e3ab6bed71
2024/07/19 22:21:07 pushed blob: sha256:f531499c6b730fc55a63e5ade55ce2c849bbf03f894248e3a2092689e3749312
2024/07/19 22:21:08 pushed blob: sha256:058cf3d8c2ba04ad7c064698c08c5e886a8623c0ad6171b8d72684253534417d
2024/07/19 22:21:10 pushed blob: sha256:7c12895b777bcaa8ccae0605b4de635b68fc32d60fa08f421dc3818bf55ee212
2024/07/19 22:21:11 pushed blob: sha256:5664b15f108bf9436ce3312090a767300800edbbfd4511aa1a6d64357024d5dd
2024/07/19 22:21:11 pushed blob: sha256:33e068de264953dfdc9f9ada207e76b61159721fd64a4820b320d05133a55fb8
2024/07/19 22:21:11 pushed blob: sha256:27be814a09ebd97fac6fb7b82d19f117185e90601009df3fbab6f442f85cd6b3
2024/07/19 22:21:14 pushed blob: sha256:4aa0ea1413d37a58615488592a0b827ea4b2e48fa5a77cf707d0e35f025e613f
2024/07/19 22:21:14 pushed blob: sha256:da7816fa955ea24533c388143c78804c28682eef99b4ee3723b548c70148bba6
2024/07/19 22:21:15 pushed blob: sha256:9aee425378d2c16cd44177dc54a274b312897f5860a8e78fdfda555a0d79dd71
2024/07/19 22:21:17 pushed blob: sha256:3bafb296168895177ec856d53d67f0fb7bf1aabeda625dcf441db11f91cc8eef
2024/07/19 22:21:18 pushed blob: sha256:acb81118b6a57035f85d451554eb802d0b1f8d55ea52298500993d7c7c94549a
2024/07/19 22:21:18 pushed blob: sha256:281a73ce7346f6fc4719c9831e40b62c544bdd9d0d789d292e261321f6b7c4a5
2024/07/19 22:21:19 pushed blob: sha256:ce670d28e15e760d7119c950c8cff8ffa8a66a12a1153bd523b4c7874296fd9f
2024/07/19 22:22:45 pushed blob: sha256:914d4d4dc529ce7809c22aff484e3cd65b5e86cd38f86c997e5b5357797a4ad8
2024/07/19 22:22:46 us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue@sha256:a2e0b9228d447f4cfbe0fd5d7cead3eae5f2e3c76c1a4a7bf26e378c4ec3c318: digest: sha256:a2e0b9228d447f4cfbe0fd5d7cead3eae5f2e3c76c1a4a7bf26e378c4ec3c318 size: 2647
2024/07/19 22:22:53 pushed blob: sha256:76056cc213d745ef36399d1ee00f602e395d7f7f8dd33932c106855bb23aaffd
2024/07/19 22:22:53 us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue@sha256:866f53b4c619d0a5f0dcccfce99ca63db72e4e83e9df51885559be68fde3b7da: digest: sha256:866f53b4c619d0a5f0dcccfce99ca63db72e4e83e9df51885559be68fde3b7da size: 2647
2024/07/19 22:22:54 pushed blob: sha256:02c4476ad2591340f072637bff6d0886dc936cb4033f124da10ed87775e62af7
2024/07/19 22:22:54 us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue@sha256:02876d2f01c8a4e6d43c264e1df21806a275769112039c54dc836c9d840f8927: digest: sha256:02876d2f01c8a4e6d43c264e1df21806a275769112039c54dc836c9d840f8927 size: 2647
2024/07/19 22:23:00 pushed blob: sha256:15d568dbb53179097ef15e2a7472ce21c492af57dfc04c91c760e04eb008bfa6
2024/07/19 22:23:01 us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue@sha256:d906dbb83f5c30d0687ecc55a04f6f3a82aaa8d56f2d17f27b0989f3644d75a2: digest: sha256:d906dbb83f5c30d0687ecc55a04f6f3a82aaa8d56f2d17f27b0989f3644d75a2 size: 2647
2024/07/19 22:23:01 us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue:v0.7.1: digest: sha256:98a2eb0e6fed19a33786b0da07332c6be2f52d07a3b2b085b369898b3b11b81e size: 1251
```

Reply:
The other images of the kueue project will be at the same level as the kueue image.

Reply:
Ok, I've updated the registry name to `us-central1-docker.pkg.dev/k8s-staging-images/kueue` so that we can reference the image as `us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue:latest`.